kdiff -urN a/doc/sample.nanorc.in b/doc/sample.nanorc.in
--- a/doc/sample.nanorc.in	2026-07-31 14:07:59
+++ b/doc/sample.nanorc.in	2026-08-01 03:00:24
@@ -77,7 +77,7 @@
 # set guidestripe 75
 
 ## Remember the used search/replace/command strings for the next session.
-# set historylog
+set historylog
 
 ## Display a "scrollbar" on the righthand side of the edit window.
 # set indicator
@@ -90,10 +90,10 @@
 
 ## Let other users (of nano or vim) know that you are editing the file
 ## they are trying to edit, and the other way around.
-# set locking
+set locking
 
 ## Fall back to slow libmagic to try and determine an applicable syntax.
-# set magic
+set magic
 
 ## The opening and closing brackets that are found by a matching-bracket
 ## search.  This may not contain blank characters.  The opening brackets
@@ -183,7 +183,7 @@
 
 ## Use the end of the title bar for some state flags: I = auto-indenting,
 ## M = mark, L = hard-wrapping long lines, R = recording, S = soft-wrapping.
-# set stateflags
+set stateflags
 
 ## Use this tab size instead of the default; it must be greater than 0.
 # set tabsize 8
@@ -254,7 +254,7 @@
 ## For all details, see 'man nanorc', section SYNTAX HIGHLIGHTING.
 
 ## To include most of the existing syntax definitions, you can do:
-# include @PKGDATADIR@/*.nanorc
+include @PKGDATADIR@/*.nanorc
 
 ## Or you can select just the ones you need.  For example:
 # include @PKGDATADIR@/html.nanorc

